docker/Dockerfile.vppagent-icmp-responder-nse

```diff
@@ -18,3 +18,6 @@ COPY --from=build /go/bin/vppagent-icmp-responder-nse /bin/vppagent-icmp-respond
 RUN rm /opt/vpp-agent/dev/etcd.conf /opt/vpp-agent/dev/kafka.conf; echo 'Endpoint: "0.0.0.0:9112"' > /opt/vpp-agent/dev/grpc.conf; echo "disabled: true" > /opt/vpp-agent/dev/linux-plugin.conf
 COPY dataplane/vppagent/conf/vpp/startup.conf /etc/vpp/vpp.conf
 COPY examples/conf/vppagent-icmp-responder-nse/supervisord.conf /etc/supervisord/supervisord.conf
+
+RUN rm -f /dev/shm/db /dev/shm/global_vm /dev/shm/vpe-api
+ENTRYPOINT ["/usr/bin/supervisord","-c","/etc/supervisord/supervisord.conf"]
```

Comment:
Could we make this CMD instead of entrypoint?

Reply:
if we want supervisord to receive signals, this is the simplest way